src/components/app/Details.js

```diff
@@ -61,6 +96,7 @@ class ProfileViewer extends PureComponent<Props> {
           tabOrder={tabOrder}
           onSelectTab={this._onSelectTab}
           onChangeTabOrder={changeTabOrder}
+          extraElements={extraButton}
```

Comment:
I went back and forth about how to insert the button in the DOM. In the end I decided to include it in the TabBar, but still define it in Details, because I thought that while it was part of the TabBar visually the TabBar itself shouldn't have to know anything about the sidebar.

Reply:
Sounds reasonable.